### PR TITLE
Add Windows testing

### PR DESCRIPTION
**Pull request by eed3si9n**
This adds rudimentary scripted testing on Windows on AppVeyor.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -310,8 +311,7 @@ lazy val commandProj = (project in file("main-command"))
   .settings(
     testedBaseSettings,
     name := "Command",
-    libraryDependencies ++= Seq(launcherInterface, sjsonNewScalaJson.value, templateResolverApi,
-      jna, jnaPlatform),
+    libraryDependencies ++= Seq(launcherInterface, sjsonNewScalaJson.value, templateResolverApi),
```
Comment (dwijnand) [inline build.sbt:314]: Why are we dropping JNA? Did we end up not using it anymore in sbt?

**Review by dwijnand — APPROVED** [review]
LGTM, just 1 question.

File: build.sbt (hunk 1 above)
Comment (eed3si9n) [reply build.sbt:314]: I am dropping it here to align JNA version with the one introduced by IO.

Comment (dwijnand) [reply build.sbt:314]: 👌

**Pull request merged**